diagram/plantuml.py

```diff
@@ -33,6 +43,7 @@ def generate(self):
             command.append("-charset")
             command.append(charset)
 
+        print("EEEE:", self.file)
```

Comment:
Is this for debugging?  Probably should remove or (at the very least) make it clearer what it means.